src/governance-classes.h

```diff
@@ -44,35 +43,24 @@ std::vector<std::string> SplitBy(std::string strCommand, std::string strDelimit)
 *   - After triggers are activated and executed, they can be removed
 */
 
-CAmount ParsePaymentAmount(const std::string& strAmount);
-
 class CGovernanceTriggerManager
 {
     friend class CSuperblockManager;
     friend class CGovernanceManager;
 
-public: // Typedefs
+private:
```

Comment:
I don't think there's any real reason to make typedef's private they don't control access to any data.